client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -134,20 +134,26 @@ const BrewRenderer = createClass({
 	},
 
 	render : function(){
-		return <div className='brewRenderer'
-			onScroll={this.handleScroll}
-			ref='main'
-			style={{ height: this.state.height }}>
-
-			<ErrorBar errors={this.props.errors} />
-			<RenderWarnings />
-
-			<div className='pages' ref='pages'>
-				{this.renderPages()}
-			</div>
-			{this.renderPageInfo()}
-			{this.renderPPRmsg()}
-		</div>;
+		return (
+			<React.Fragment>
+				<div className='brewRenderer'
+					onScroll={this.handleScroll}
+					onMouseOver={this.prepareScroll}
```

Comment:
We don't need this (and `onMouseOut`) any more, right? In particular, the functions no longer exist...